server/test/handlers/inputSchedule.test.js

```diff
@@ -1,4 +1,6 @@
-const timeslotHandler = require('../../models/handlers/timeslot');
+const timeslotHandler = require('../../models/handlers/workerTimeslot');
+
+//const timeslotHandler = require('../models/handlers/timeslot');
```

Comment:
Hi Peter,
If this is no longer needed, I would recommend removing it (or adding a comment about why it's kept) to improve code readability.

Reply:
Resolved! Good catch